sapl/norma/models.py

```diff
@@ -194,8 +195,9 @@ def save(self, force_insert=False, force_update=False, using=None,
 class NormaEstatisticas(models.Model):
     usuario = models.CharField(max_length=50)
     horario_acesso = models.DateTimeField(
-        blank=True, null=True,
-        auto_now=True)
+        blank=True, null=True)
+    ano = models.PositiveSmallIntegerField(verbose_name=_('Ano'),
+                                           choices=RANGE_ANOS, default=datetime.date.today().year)
```

Comment:
Existe um probleminha aqui, _se não me engano_. Quando o default faz uma referência a uma data, a data é avalidada na hora que o model é definido e não quando o registro é criado. Vide aqui: https://stackoverflow.com/a/2771701/2698109

Neste caso, segundo esse link https://stackoverflow.com/questions/1517474/only-showing-year-in-django-admin-a-yearfield-instead-of-datefield vc poderia colocar assim `default=lambda: datetime.datetime.now().year` pois daí quando adicionar um novo registro o python chama a função lambda que pega o ano corrente.